web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -241,6 +249,23 @@ export default {
     ...mapGetters('dandiset', ['version']),
   },
   asyncComputed: {
+    userCanModifyDandiset: {
+      async get() {
+        if (this.user.admin) {
+          return true;
+        }
+
+        const { identifier } = this.publishDandiset.meta.dandiset;
+        const { data: owners } = await publishRest.owners(identifier);
+        const userExists = owners.find((owner) => owner.username === this.user.username);
+        if (userExists !== undefined) {
+          return true;
+        }
+
+        return false;
```

Comment:
return !!
```suggestion
        return !!userExists;
```
I'm never sure if JS hacks like this really work, but it's shorter.

Reply:
Actually in this case I think the right thing to do is use [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) directly, instead of `.find()`.

Reply:
`includes` only works when searching for a direct match in a list (i.e. you'd do something like `owners.includes(this.user)`. In this case, since we're trying to match objects, this will fail, since equivalent objects don't match with a simple `===` check.

Reply:
Bless javascript's little heart.